### ISSUE TITLE
Unit tests for strategizer

### ISSUE THREAD

**Issue by mmandlis** [PolymerLabs/arcs#680]

**Comment by shaper** [comment]
There are a batch of tests under `runtime/test/strategies` now. But maybe this was about adding unit tests for the strategizer at a level above the individual strategies?

**Comment by mmandlis** [comment]
yes, this is for tests for the strategizer itself, not strategies. 
the individual strategies have reasonable test coverage, i think.
